src/main/java/io/pinecone/configs/PineconeConfig.java

```diff
@@ -70,21 +72,23 @@ public PineconeConfig(String apiKey) {
      * @param sourceTag An optional source tag to be included in the user agent.
      */
     public PineconeConfig(String apiKey, String sourceTag) {
-        this(apiKey, sourceTag, null);
+        this(apiKey, sourceTag, null, null);
     }
 
     /**
-     * Constructs a {@link PineconeConfig} instance with the specified API key, source tag, control plane proxy
-     * configuration, and data plane proxy configuration.
+     * Constructs a {@link PineconeConfig} instance with the specified API key, source tag, a HTTP proxy configuration,
```

Comment:
customOkHttpClient is now a member of PineconeConfig. It makes a lot of sense since gRPC custom channel is already a part of this config class.